tests/JsonSchemaProvider.Tests/JsonSchemaProvider.Tests.fsproj

```diff
@@ -1,23 +1,34 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net8.0</TargetFramework>
     <GenerateProgramFile>false</GenerateProgramFile>
   </PropertyGroup>
   <ItemGroup>
+    <None Include="Debug.fsx" />
     <Compile Include="SchemaConversionTests.fs" />
     <Compile Include="JsonSchemaProviderTests.fs" />
     <Compile Include="Main.fs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="altcover" Version="8.8.53" />
+    <PackageReference Update="FSharp.Core" Version="8.*" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="altcover" Version="8.9.3" />
     <PackageReference Include="Expecto" Version="10.*" />
-    <PackageReference Include="YoloDev.Expecto.TestSdk" Version="0.*" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.*" />
-    <PackageReference Update="FSharp.Core" Version="8.*" />
+    <PackageReference Include="YoloDev.Expecto.TestSdk" Version="0.*" />
+    <PackageReference Include="FSharp.Data.Json.Core" Version="6.4.0" />
+    <PackageReference Include="NJsonSchema" Version="11.0.2" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\src\JsonSchemaProvider.Runtime\JsonSchemaProvider.Runtime.fsproj" />
-    <ProjectReference Include="..\..\src\JsonSchemaProvider.DesignTime\JsonSchemaProvider.DesignTime.fsproj" />
+    <Reference Include="JsonSchemaProvider">
```

Comment:
@xperiandri: I had to remove the assembly references here and replace them by project references again since that broke the build with `./build.sh DotnetBuild`, e.g. This might make trouble ein Visual Studio again, something I cannot test since I don't have it as it is discontinued for macOS.

Reply:
I would've commented that but OK

Reply:
@xperiandri, you mean comment the code to reestablish it quickly if needed?
Or do you mean a comment here in GitHub before the merge of your PR?
The former, I could do, sure.
Regarding the latter: I just noticed the build failure after I merged.

Reply:
I meant commenting within `fsproj` file to be able to uncomment for development purposes

Reply:
I re-added the lines as commented elements.